tests/mpls_test.py

```diff
@@ -0,0 +1,91 @@
+import os
+import sys
+import traceback
+from unittest import mock
+
+from click.testing import CliRunner
+
+from .mock_tables import dbconnector
+
+import config.main as config
+import show.main as show
+from utilities_common.db import Db
+
+show_interfaces_mpls_output="""\
+Interface     MPLS State
+------------  ------------
+Ethernet2     enable
+Ethernet4     disable
+Ethernet8     disable
+Ethernet16    disable
+Loopback0     disable
+PortChannel2  disable
+Vlan2         enable
+"""
+
+show_interfaces_mpls_specific_output="""\
+Interface    MPLS State
+-----------  ------------
+Ethernet2    enable
+"""
+
+modules_path = os.path.join(os.path.dirname(__file__), "..")
+test_path = os.path.join(modules_path, "tests")
+sys.path.insert(0, modules_path)
+sys.path.insert(0, test_path)
+mock_db_path = os.path.join(test_path, "mpls_input")
+
+
+class TestMpls(object):
+    @classmethod
+    def setup_class(cls):
+        print("SETUP")
+        os.environ['UTILITIES_UNIT_TESTING'] = "1"
+
+    def test_config_mpls_add(self):
+        runner = CliRunner()
+        db = Db()
+        obj = {'config_db':db.cfgdb}
+
+        result = runner.invoke(config.config.commands["interface"].commands["mpls"].commands["add"], ["Ethernet4"], obj=obj)
+        print(result.exit_code)
+        print(result.output)
+        assert result.exit_code == 0
```

Comment:
Thanks for adding the tests. However the test is not testing the actual update in the Db. Successful result code does not mean the config change has happened, you should test for the config DB change to ensure the config change.

Reply:
@smahesh this is similar to other tests that I have seen throughout sonic-utilities/tests.  None of them check CONFIG_DB to see if it is updated.  I looked at testcases in config_int_ip_test.py, vxlan.py, portchannel.py, among others.  They check the return code and that is it.

Reply:
Those tests were added before the ability to test config DB changes. All the new CLIs should test for the config DB updates, not just the return status.

Reply:
Other option is add this as part of VS tests (sonic-swss) where configuring MPLS will also update the APP_DB.

Reply:
@smaheshm mpls_test.py has been updated with CONFIG_DB check.